pkgs/development/python-modules/asap3/default.nix

```
@@ -0,0 +1,34 @@
+{
+  pkgs,
```

`pkgs.callPackage` already allows all `pkgs` for its top args
```suggestion
  which,
```